substrate/frame/nomination-pools/src/lib.rs

```diff
@@ -2935,9 +2977,12 @@ pub mod pallet {
 			);
 
 			let pool_contribution = member.total_balance();
-			ensure!(pool_contribution >= MinJoinBond::<T>::get(), Error::<T>::MinimumBondNotMet);
-			// the member must have some contribution to be migrated.
-			ensure!(pool_contribution > Zero::zero(), Error::<T>::AlreadyMigrated);
+			// ensure the pool contribution is greater than the existential deposit otherwise we
+			// cannot transfer funds to member account.
+			ensure!(
```

Comment:
Remove this!!

Reply:
shouldn't be removed as transferring amount lower than ED fails. But this does not happen for any existing pool member on polkadot or kusama so it is only defensive.